src/Opserver.Web/ContentSource/js/Scripts.js

```diff
@@ -1445,12 +1447,41 @@ Status.Exceptions = (function () {
             .get();
     }
 
+    function getHostFilterValue() {
+        return $('#hostFilterInput').val();
+    }
+
+    function getUrlFilterValue() {
+        return $('#urlFilterInput').val();
+    }
+
     $(document).on('submit', '#exceptionFiltersForm', function (event) {
-        let selectedLogLevels = getSelectedLogLevels();
         let searchParams = new URLSearchParams(window.location.search);
-        searchParams.set("logLevels", selectedLogLevels.join(','));
+        let selectedLogLevels = getSelectedLogLevels();
+        if (selectedLogLevels.length === 0) {
+            searchParams.delete("logLevels");
+        } else {
+            searchParams.set("logLevels", selectedLogLevels.join(','));
+        }
+        let host = getHostFilterValue();
+        if (host === null || host === '') {
```

Comment:
In the QA portion of my brain I wonder if a space will succeed here. Not a big deal because the search will come up empty and they'll try again, but it might be confusing when the search field _looks_ empty.

Reply:
I agree with @AaronBertrand - we should sanitize these inputs to ensure that random whitespace doesn't make its way in.

Reply:
Make sense! I've sanitazed the inputs on both frontend and backend with a trim.